opal/corelib/runtime.js

```diff
@@ -2299,10 +2299,6 @@
     for (i = 0, l = paths.length; i < l; i++) {
       path = Opal.normalize(paths[i]);
 
-      if (Opal.require_table[path]) {
-        continue;
```

Comment:
I don't think we can remove this code.
The above code:
```js
for (i = 0; i < 3; i++) { 
  if (i === 1) {
    continue;
  }
  console.log(i);
}
// print 0,2
```
is *not* equivalent to:

```js
for (i = 0; i < 3; i++) {
  console.log(i);
}
// print 0,1,2
```

In this case we dont want to push the same path in `Opal.loaded_features`.

Reply:
Oh crap, i was completely betriebsblind, sorry